torch/optim/swa_utils.py

```diff
@@ -26,8 +28,11 @@ class AveragedModel(Module):
             :class:`AveragedModel` parameter, the current value of :attr:`model`
             parameter and the number of models already averaged; if None,
             equally weighted average is used (default: None)
-        mode (str, optional): whether to use ``'parameters'`` or ``'state_dict'`` for update
-            (default: ``'parameters'``)
+        use_buffers (bool, optional): If the model has layers that contain
+            buffers(like BN), passing `use_buffers=True` would use buffers
+            along with parameters to perform the update. This has shown to
```

Comment:
nit: "perform the update" is not super clear? Maybe just say that buffers also get averaged alongside parameters.

Reply:
@albanD You caught us in the middle of updating the documentation. :) Could you have another look?

Reply:
Sure no problem, it is a draft so I was early haha